README.md

```diff
@@ -9,37 +9,53 @@ open source packages, container images, and CI/CD configurations to identify mis
 ## Example usage for IaC and SCA
 
 ```yaml
-on: [push]
+name: checkov
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main", "master" ]
+  pull_request:
+    branches: [ "main", "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  checkov-job:
+  # This workflow contains a single job called "scan"
+  scan:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+      
+    # The type of runner that the job will run on
     runs-on: ubuntu-latest
-    name: checkov-action
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@master
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
 
-      - name: Run Checkov action
-        id: checkov
-        uses: bridgecrewio/checkov-action@master
+      - name: Checkov GitHub Action
+        uses: bridgecrewio/bridgecrew-action@master
+        
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@v2
+        
+        # Results are generated only on a success or failure
+        # this is required since GitHub by default won't run the next step
+        # when the previous one has failed. Security checks that do not pass will 'fail'.
+        # An alternative is to add `continue-on-error: true` to the previous step
+        # Or 'soft_fail: true' to checkov.
+        if: success() || failure()
         with:
-          directory: example/
```

Comment:
Why delete these? They're helpful to understand the options

Reply:
I removed these in first example in order for the snippet to be functional "out of the box" with direct copy/paste. Also why I highlighted the next section as example of using 'with' block

Reply:
The next section is specifically image scanning that has a different syntax. You could add a third for a copy/paste ready example if that's the intention

Reply:
added as third example (top of page), left original two in-tact

Reply:
Looks good!